test/bricks-net-url.test.cc

```diff
@@ -0,0 +1,261 @@
+/*******************************************************************************
```

Comment:
Any reason you kept this file? Perhaps a symlink to `../../Bricks/net/url/test.cc` then?